components/style/properties.mako.rs

```diff
@@ -2461,7 +2469,8 @@ pub mod longhands {
     </%self:single_keyword_computed>
 
     // TODO(pcwalton): `full-width`
-    ${single_keyword("text-transform", "none capitalize uppercase lowercase")}
+    ${single_keyword("text-transform", "none capitalize uppercase lowercase",
+                     extra_gecko_values="full-width")}
```

Comment:
Just add it to the whitelist in geckolib/properties.mako.rs with a comment saying what needs to be fixed. There are several others of these, and I'll do a bulk patch at done point to fix then in gecko.